app/models/camp.rb

```diff
@@ -84,6 +85,10 @@ class Camp < ApplicationRecord
       )
     }
 
+  scope :with_tags, lambda { |tag|
```

Comment:
This one feels a bit odd, why not just use `tagged_with` as an available filter?

Reply:
Can you clarify? I don't truly understand what is and what is not possible with rails/ruby. Just replicating other existing code.